covid19-dashboard/src/Config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "host": "http://localhost/",
+  "geoIdParam": "geoId",
+  "placeTypeParam": "placeType",
+  "SUBTITLE": "COVID-19 Dashboard",
+  "FOOTER": "Data from WHO & NYT, based on reports from state and local health agencies",
+  "placeTypes": ["Country", "State", "County"],
+  "subregionSelectionButton": [{
```

Comment:
my reasoning for this is that the dashboard is supposed to work on different stat_vars, so different stat_vars would require different subtitle and names. For example, I am planning on doing a Poverty dashboard next.

Reply:
Ok, though I find it's usually better to make something configurable as you need it to be (it could turn out to add unnecessary complexity if unused).